docs/usage/self-hosted-configuration.md

```diff
@@ -895,6 +923,15 @@ It also may mean that ignored directories like `node_modules` can be preserved a
 
 ## platform
 
+## platformVersion
+
+Set a string here to let Renovate skip some API calls to fetch the platform version.
+Renovate will use this value as the GitLab or Bitbucket server version.
+
+Particularly useful with GitLab's `CI_JOB_TOKEN` to authenticate Renovate or to reduce API calls for Bitbucket.
+
+Read [platform details](modules/platform/gitlab/index.md) to learn why we need the server version on GitLab.
```

Comment:
Is this the correct format for the link? I usually mess this up, but often the links have `./` or `../` in front of them.

Reply:
It works fine in the [live docs](https://docs.renovatebot.com/self-hosted-experimental/#renovate_x_platform_version)